index.html

```diff
@@ -326,8 +326,10 @@ <h2><dfn>observe()</dfn> method</h2>
         <p>The <a>observe()</a> method instructs the user agent to
         <dfn>register the observer</dfn> and must run these steps:</p>
         <ol data-link-for="PerformanceObserverInit">
-          <li>Let <var>entry types</var> be <var>options</var> <a>entryTypes</a>
+          <li>Let <var>entry types</var> be <var>options</var>' <a>entryTypes</a>
             sequence.</li>
+          <li>For each dictionary of name <var>entryType</var> in <var>options</var>,
+            add the string `<var>entryType</var>` to <var>entry types</var>.
```

Comment:
Not sure I grok what this is doing. What is an example of "each dictionary of name `entryType`"? :-)

I _think_ you're enumerating the keys in options and adding them to entryTypes? As an aside, can we actually do this without first cloning the options? As in, I don't think we should be modifying the passed in object.

Reply:
Yes, maybe enumarating keys in options is clearer? Or enumerating keys whose values are dictionaries (to exclude entryTypes itself for example)?

Reply:
Per later comment, I don't think we should be modifying this object in place.. But if we're cloning it first, we could: pluck entryTypes out first and `<do-stuff>`, then enumerate the remaining keys and `<do-more-stuff>`, arriving at a consistent internal object that we can use later in the algorithm?

Reply:
Yes, so I propose adding something like "Each PerformanceObserver gets an associated |entry types| object". Then here we would modify observer's |entry types| instead of some variable of option's entryTypes array. Does that sound reasonable?